kogito-editors-js/docs/new-component.md

```diff
@@ -0,0 +1,105 @@
+# How to create a Kogito editor React component
+
+All Kogito editor React components live in the `packages` directory and follow the convention of **components** and **loaders**.
+
+- **Components** - all elements an editor may be a marshaller, a dialog, a Boxed Expression editor, or even a FEEL editor.
+
+- **Loaders** - this is more specific. It teaches how each editor (DMN or BPMN) may load their components.
+
+In this guide, we will create a **component** and add it to GWT in 5 steps.
+
+### 1. Create a new component
+
+Let's use the following script to create the minimal boilerplate code that a React component needs:
+
+```bash
+yarn new-component
+```
+
+Once you execute this, insert the name of your new component. In my case, In my case, I will call it `feel-dialog`.
+
+### 2. Run the development web app
+
+As I named my component as `feel-dialog`, I type this command to access it:
+
+```bash
+cd packages/feel-dialog-component
+```
+
+> **Notice**: The directory already has the proper suffix by convention!
+
+Now you may run this to install the default dependencies of your component and development web app:
+
+```bash
+yarn install
+
+cd showcase
+yarn install
+```
+
+Finally, you're ready to run the web app and see your component alive:
+
+```
+yarn start
+```
+
+### 3. Expose with the a Loader
+
+Now, we need to get this component in some loader. So, your GWT editor (DMN or BPMN) will be able to render it.
+
+In this example, I will use the `dmn-loader`. Thus, we may:
+
+- Open the `packages/dmn-loader` directory
+- Add the `feel-dialog-component` in the `package.json` as a dependency
+- And, finally, add the following snippet in the `src/index.tsx` file:
+
+```typescript
+import { FeelDialog } from "feel-dialog-component";
+
+const renderFeelDialog = (selector: string) => {
+  ReactDOM.render(<FeelDialog />, document.querySelector(selector));
+};
+
+export { renderFeelDialog };
```

Comment:
I am getting this error at this point
![Screenshot from 2021-07-02 09-25-42](https://user-images.githubusercontent.com/8044780/124237180-89f1fb80-db17-11eb-8fd9-48637ca77a59.png)

Reply:
Fix applied :-) Thanks!